tests/Zend/Log/Formatter/SimpleTest.php

```diff
@@ -84,11 +84,11 @@ function testComplexValues()
 
         $fields['message'] = 10;
         $line = $f->format($fields);
-        $this->assertContains($fields['message'], $line);
+        $this->assertContains((string) $fields['message'], $line);
```

Comment:
Newer version of PHPUnit requires that this value is a string in this case (since second parameter is a string). Casting to string.